tools/ckpts/convert_hf_to_sequential.py

```diff
@@ -519,15 +519,15 @@ def get_non_existing_dir(tmp_dir):
     model, optimizer, _, lr_scheduler = deepspeed.initialize(
         model=model,
         optimizer=optimizer,
-        args=neox_args,
+        # args=neox_args,
```

Comment:
Why was this done?

Reply:
An issue I had if you kept both args and config_params that led me to the same change, is that DeepSpeed would throw an error about being passed two sets of configs in `DeeperSpeed/deepspeed/__init__.py` : 
```
    if hasattr(args, "deepspeed_config") and args.deepspeed_config is not None:
        assert config is None, "Not sure how to proceed, we were given deepspeed configs in the deepspeed arguments and deepspeed.initialize() function call"
```

Reply:
Ah yes this is a requirement in newer DeepSpeed. We can take this in then.